native/effektio/src/platform/android.rs

```diff
@@ -1,30 +1,101 @@
-use android_logger::{Config, FilterBuilder};
+use android_logger::{AndroidLogger, Config, FilterBuilder};
 use anyhow::Result;
-use log::Level;
+use log::{Level, LevelFilter, Log, Metadata, Record};
 use matrix_sdk::ClientBuilder;
+use std::{
+    fs,
+    path::PathBuf,
+    sync::{Arc, Mutex},
+};
 
 use super::native;
 
+pub use super::native::sanitize;
+
 pub async fn new_client_config(base_path: String, home: String) -> Result<ClientBuilder> {
     let builder = native::new_client_config(base_path, home)
         .await?
         .user_agent(format!("effektio-android/{:}", env!("CARGO_PKG_VERSION")));
     Ok(builder)
 }
 
-pub fn init_logging(filter: Option<String>) -> Result<()> {
+pub fn init_logging(log_dir: String, filter: Option<String>) -> Result<()> {
     std::env::set_var("RUST_BACKTRACE", "1");
-
     log_panics::init();
 
+    let log_level = match filter {
+        Some(ref filter) => FilterBuilder::new().parse(&filter).build(),
+        None => FilterBuilder::new().build(),
+    };
+
     let mut log_config = Config::default()
-        .with_min_level(Level::Trace)
-        .with_tag("effektio-sdk");
+        .with_max_level(LevelFilter::Trace)
+        .with_tag("com.example.effektio");
```

Comment:
we should probably move that into a const somewhere, no?

Reply:
Fixed by https://github.com/acterglobal/a3/pull/488/commits/be70dcb45f7585bfa05e325db7e4615e2ea72984